### PR TITLE
chore: fix typos, grammar, and formatting in Fabric components guide

### PR DESCRIPTION
**Pull request by coolsoftwaretyler**
Hey folks,

I have been reading through the [Fabric components guide](https://reactnative.dev/docs/fabric-native-components-introduction) and ran into some typos and grammar errors. Figured I'd post a PR with some edits.

There are two code-snippet changes that seemed incorrect to me as well. For the most part, these diffs are typos and grammar. I will drop inline comments on the items I think were incorrect to begin.

### CODE REVIEW

File: docs/fabric-native-components.md [hunk 1]
```diff
@@ -128,7 +146,7 @@ The specification is used by the React Native's Codegen tools to generate platfo
      "type": "components",
      "jsSrcsDir": "specs",
      "android": {
-       "javaPackageName": "com.nativelocalstorage"
+       "javaPackageName": "com.webview"
```
Comment (coolsoftwaretyler) [inline docs/fabric-native-components.md:149]: This seemed like an inaccuracy in the original post, but let me know if it should be `com.nativelocalstorage` for some reason. Thank you!

File: docs/fabric-native-components.md [hunk 2]
```diff
@@ -76,6 +76,23 @@ export interface NativeProps extends ViewProps {
...
+
+export interface NativeProps extends ViewProps {
+  sourceURL?: string;
+  onScriptLoaded?: BubblingEventHandler<WebViewScriptLoadedEvent> | null;
+}
+
+export default codegenNativeComponent<NativeProps>(
+  'CustomWebView',
+) as HostComponent<NativeProps>;
+
```
Comment (coolsoftwaretyler) [inline docs/fabric-native-components.md:95]: Added a newline at the end of file to satisfy the linter that comes with React Native community CLI

File: docs/fabric-native-components-ios.md [hunk 3]
```diff
@@ -212,10 +212,10 @@ Update the `AppDelegate.mm` to make your application aware of our custom WebView
 #import "AppDelegate.h"
 
 #import <React/RCTBundleURLProvider.h>
+// highlight-start
 #import <React/RCTBridge+Private.h>
-// highlight-next-line
 #import "RCTWebView.h"
-
+// highlight-end
```
Comment (coolsoftwaretyler) [inline docs/fabric-native-components-ios.md:218]: I had to add the `RCTWebView` header on my machine, so I moved the highlight to encapsulate it.

File: docs/fabric-native-components-android.md [hunk 4]
```diff
@@ -341,7 +341,7 @@ Finally, if the component can emit an event, you need to map the event name by o
 
 ### 4. Write the `ReactWebViewPackage`
 
-As you do with Native Modules, also Native Components needs to implement the `ReactPackage` class. This is an object that you can use to register the component in the React Native runtime.
+As you do with Native Modules, Native Components also needto implement the `ReactPackage` class. This is an object that you can use to register the component in the React Native runtime.
```
Comment (cortinico) [inline docs/fabric-native-components-android.md:344]:
```suggestion
As you do with Native Modules, Native Components also need to implement the `ReactPackage` class. This is an object that you can use to register the component in the React Native runtime.
```

File: docs/fabric-native-components.md (hunk 1 above)
Comment (cortinico) [reply docs/fabric-native-components.md:149]: nope it should be `com.webview` after we moved everything over to use WebViews

File: docs/fabric-native-components.md [hunk 5]
```diff
@@ -76,6 +76,23 @@ export interface NativeProps extends ViewProps {
...
+};
+
+export interface NativeProps extends ViewProps {
+  sourceURL?: string;
+  onScriptLoaded?: BubblingEventHandler<WebViewScriptLoadedEvent> | null;
+}
+
+export default codegenNativeComponent<NativeProps>(
+  'CustomWebView',
+) as HostComponent<NativeProps>;
```
Comment (cortinico) [inline docs/fabric-native-components.md:94]: This is a copypasta? Please revert

**Review by cortinico — CHANGES_REQUESTED** [review]

**Comment by cortinico** [comment]
@coolsoftwaretyler Thank you very much!

File: docs/fabric-native-components.md (hunk 5 above)
Comment (coolsoftwaretyler) [reply docs/fabric-native-components.md]: Done!

**Comment by coolsoftwaretyler** [comment]
Thanks for the review, @cortinico. I think I've resolved all comments. I appreciate your time!

**Review by cortinico — APPROVED** [review]

**Pull request merged**